dev/ci/README.md

```
@@ -205,9 +205,205 @@ to gate. The exception is `asan`, which does fail.
+not finish at all: it ran 09:33:03 -> 11:03:20, hit the 90 min cap, and its
+log ends `Terminate orphan process: pid (4681) (CatchTestRunner)`.
+
+### Why `verbosity=1` is not set
+
+The job used to run with `ASAN_OPTIONS=verbosity=1:...`. That was removed, and
+if you are tempted to put it back: it costs a great deal and buys nothing.
+
+At `verbosity>=1` ASan reports its own container-annotation bookkeeping, which
+for this Eigen- and `std::vector`-heavy suite is a sustained stream of
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)